tutorials/kotlin-springboot-app-engine/index.md

```diff
@@ -99,16 +99,24 @@ To deploy your application, you will use an App Engine plugin for Maven which si
 is also [available for Gradle](https://cloud.google.com/appengine/docs/standard/java/tools/gradle).
 
 
-1.  Create a file called `app.yaml` in a new folder `src/main/appengine` with the following contents:
-
-        runtime: java
-        env: flex
-        runtime_config:
-          jdk: openjdk8
-
-    By specifying `runtime: java`, the runtime image `gcr.io/google-appenine/openjdk:8` is automatically selected
-    when you deploy a JAR (*.jar) file. The JDK version is also selected using the `jdk` field.
-
+1.
+   _When running on JDK 8:_ 
+   
+   Create a file called app.yaml in a new folder src/main/appengine with the following contents:
```

Comment:
In the original, this filename is surrounded by backticks to put it in code font. That needs to be retained in your modification. The same applies to the other filenames and paths.

Reply:
added the back ticks